src/main/java/com/transferwise/openbanking/client/api/vrp/RestVrpConsentClient.java

```diff
@@ -0,0 +1,215 @@
+package com.transferwise.openbanking.client.api.vrp;
+
+import com.transferwise.openbanking.client.api.common.AuthorizationContext;
+import com.transferwise.openbanking.client.api.common.BasePaymentClient;
+import com.transferwise.openbanking.client.api.common.IdempotencyKeyGenerator;
+import com.transferwise.openbanking.client.api.common.OpenBankingHeaders;
+import com.transferwise.openbanking.client.api.payment.v3.model.vrp.OBDomesticVRPConsentRequest;
+import com.transferwise.openbanking.client.api.payment.v3.model.vrp.OBDomesticVRPConsentResponse;
+import com.transferwise.openbanking.client.api.payment.v3.model.vrp.OBDomesticVRPRequest;
+import com.transferwise.openbanking.client.api.payment.v3.model.vrp.OBVRPFundsConfirmationRequest;
+import com.transferwise.openbanking.client.api.payment.v3.model.vrp.OBVRPFundsConfirmationResponse;
+import com.transferwise.openbanking.client.configuration.AspspDetails;
+import com.transferwise.openbanking.client.configuration.SoftwareStatementDetails;
+import com.transferwise.openbanking.client.error.ApiCallException;
+import com.transferwise.openbanking.client.json.JsonConverter;
+import com.transferwise.openbanking.client.jwt.JwtClaimsSigner;
+import com.transferwise.openbanking.client.oauth.OAuthClient;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestClientResponseException;
+import org.springframework.web.client.RestOperations;
+
+@Slf4j
+public class RestVrpConsentClient extends BasePaymentClient implements VrpConsentClient {
```

Comment:
Can wee merge these two `VrpConsentClient` and `VrpClient` right now its a bit confusing from name which one to use according to use case.

Reply:
As we store a separate client bean for each bank it makes sense to merge them. Will do